dex/networks/eth/txdata_test.go

```diff
@@ -18,16 +22,60 @@ func mustParseHex(s string) []byte {
 	return b
 }
 
+func initiationsAreEqual(i1, i2 ETHSwapInitiation) bool {
+	return i1.RefundTimestamp.Cmp(i2.RefundTimestamp) == 0 &&
+		bytes.Equal(i1.SecretHash[:], i2.SecretHash[:]) &&
+		bytes.Equal(i1.Participant.Bytes(), i2.Participant.Bytes()) &&
```

Comment:
You can also just do:
```
...
i1.SecretHash == i2.SecretHash &&
i1.Participant == i2.Participant &&
...
```
Because they are arrays and not slices.

Also not important, but using numerals after variable names is kind of confusing to me. I would prefer maybe just a, b, c, ... but it's probably just a style thing. `1` in particular can be confusing as a variable name because it kind of looks like an upper case I or a lower case l or the numeral 1 or even or |.